templates/status.html

```diff
@@ -14,63 +14,85 @@
+                {% endif %}
+              </time>
+            {% /description_item %}
+            {% #description_item title="Job requests not yet processed by "|add:backend.name|add:" backend" stacked=True %}
+              {{ backend.queue.unacked|floatformat:"-3g" }}
+            {% /description_item %}
+            {% #description_item title="Total job requests received by "|add:backend.name|add:" backend" stacked=True %}
+              {{ backend.queue.acked|floatformat:"-3g" }}
+            {% /description_item %}
+          </div>
+          {% #card_footer no_container class="border-t border-t-slate-200" %}
+            <p class="text-sm font-medium text-slate-600">
+              {{ backend.name }} backend is missing if OpenSAFELY Jobs does
+              not receive a response for {{ backend.alert_timeout|duration }}.
+            </p>
+          {% /card_footer %}
+        {% /card %}
+      {% endfor %}
+    </div>
+
+
+    <h2 id="db-maintenance-mode">Database maintenance mode</h2>
+    <p>To ensure you're always working with the most up-to-date data, the database undergoes routine maintenance once a week. During this time, the system will be temporarily unavailable while we update the data.
+
+      <details>
+        <summary>What this means for you</summary>
         <div>
-          {% #description_item title="Running jobs" stacked=True %}
-            {{ backend.queue.running }}
-          {% /description_item %}
-          {% #description_item title="Pending jobs" stacked=True %}
-            {{ backend.queue.pending }}
-          {% /description_item %}
-          {% #description_item title="Last contact with "|add:backend.name|add:" backend" stacked=True %}
-            <time
-              class="
-                     text-slate-900 group relative cursor-pointer
-                     {% if not backend.last_seen %}text-bn-ribbon-900{% endif %}
-                    "
-              datetime="{{ backend.last_seen|date:"Y-m-d H:i:sO" }}">
-              {{ backend.last_seen|naturaltime }}
-              {% if backend.last_seen %}
-                {% tooltip class="font-mono" position="-bottom-4" content=backend.last_seen|date:"d M Y H:i:s e" %}
-              {% endif %}
-            </time>
-          {% /description_item %}
-          {% #description_item title="Job requests not yet processed by "|add:backend.name|add:" backend" stacked=True %}
-            {{ backend.queue.unacked|floatformat:"-3g" }}
-          {% /description_item %}
-          {% #description_item title="Total job requests received by "|add:backend.name|add:" backend" stacked=True %}
-            {{ backend.queue.acked|floatformat:"-3g" }}
-          {% /description_item %}
+          <ul>
+            <li>Database jobs will be paused during maintenance.</li>
+            <li>Non-database jobs will still run, but they will be run against previously extracted data, which may not reflect the most current data in the database.</li>
```

Comment:
To be pedantic, non-database jobs always run against previously extracted data (i.e the data was extracted from the last run of a db job). It's probably more accurate to say that non-db jobs will be unaffected, unless they are dependent on a database job that's been paused due to maintenance mode, so it might take longer than normal to run.